src/CircuitBreaker/src/Hystrix.MetricsEventsCore/HystrixServiceCollectionExtensions.cs

```diff
@@ -32,50 +37,67 @@ public static void AddHystrixMetricsEventSource(this IServiceCollection services
                 throw new ArgumentNullException(nameof(services));
             }
 
+            services.AddHystrixMetricsStream();
```

Comment:
Added this line (and used TryAdd above) because `HystrixEventSourceService` requires `HystrixDashboardStream`, so this should help avoid an exception without causing any problems